nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees_desc_asb_[7,).partial.md

```diff
@@ -1,19 +1,32 @@
-In NServiceBus version 6 and above transports fully control the transactional support. A transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The Azure Service Bus transport only uses a transaction scope to implement the `SendAtomicWithReceive` transaction mode. Other modes work without using transaction scope. 
+NOTE: This is advanced documentation and only of significant relevance when developing custom behaviors or satellites. To make a long story short: As from version 7, transactional operations are enlisted in the receive scope, if present (only in `SendAtomicWithReceive` mode). And all transactional operations performed in the pipeline must be wrapped by a suppress scope. Read on to learn why.
```

Comment:
How about we say put `SendAtomicWithReceive` more in focus and say something like:

> When running in `SendAtomicWithReceive` ASB transport opens transaction scope that cannot be enlisted by any other resource manager. As a result any custom behaviors or satellites need to make an explicit transaction scope suppress.